README.md

```diff
@@ -12,11 +12,11 @@ pip install git+https://github.com/Imageomics/cautious-robot
 
 ## How it Works
 
-Cautious-robot will check the provided CSV for `IMG_NAME`, `URL`, and `SUBFOLDERS` (if provided), then download all images that have a value in the `IMG_NAME` column. Note that choice of image filename should be unique, cautious-robot will refuse the request if the filename column selected is not a unique identifier for the dataset. Images that have a filename but no `URL` are recorded in the error log. Logs are saved in the same directory as the source CSV. Please note that images may be not be logged in errors if the stream is interrupted, but the verifier would register them as missing.
+Cautious-robot will check the provided CSV for `IMG_NAME`, `URL`, and `SUBFOLDERS` (if provided), then download all images that have a value in the `IMG_NAME` column. Note that choice of image filename should be unique, cautious-robot will refuse the request if the filename column selected is not a unique identifier for the dataset. It will also check if the provided `OUTPUT` folder already exists, asking the user before proceeding. Images that have a filename but no `URL` are recorded in the error log; the user is prompted to ignore or address the missing URLs prior to downloading. Logs are saved in the same directory as the source CSV (logging is done by adding to an existing JSON, so it will not overwrite existing logs with the same name in case of a restarted download). Please note that images may be not be logged in errors if the stream is interrupted, but the verifier would register them as missing.
```

Comment:
```suggestion
Cautious-robot will check the provided CSV for `IMG_NAME`, `URL`, and `SUBFOLDERS` (if provided), then download all images that have a value in the `IMG_NAME` column. Note that choice of image filename should be unique; cautious-robot will refuse the request if the filename column selected is not unique within the dataset. It will also check if the provided `OUTPUT` folder already exists, asking the user before proceeding. Images that have a filename but no `URL` are recorded in the error log; the user is prompted whether to ignore or address the missing URLs prior to downloading. Logs are saved in the same directory as the source CSV (logging is done by adding to an existing JSON, so it will not overwrite existing logs with the same name in case of a restarted download). Please note that images may not be logged in errors if the stream is interrupted, but the verifier would register them as missing.
```

Reply:
I'm not sure what you mean by _logged in errors_. Is it _may not be logged in the error log in case of errors_ or something else?

Reply:
Yes, I meant it _may not be logged in the error log_.